include/Track.h

```diff
@@ -146,6 +146,8 @@ class TrackContentObject : public Model, public JournallingObject
 		return m_selectViewOnCreate;
 	}
 
+	/// Returns true if and only if a->startPosition() < b->startPosition()
+	static bool comparePosition(const TrackContentObject* a, const TrackContentObject* b);
```

Comment:
This is only needed inside `Song` and uses a public `TrackContentObject` function. It should a static function in `Song.cpp`.